docs/source/beersheba.rst

```diff
@@ -69,7 +69,7 @@ Besides the :ref:`Common arguments to every city`, *Beersheba* has the following
      - Sampling size of the sensors (usually the pitch) in *mm*.
 
    * - ``inter_method``
-     - ``str`` (**None**, '**linear**', '**cubic**')
+     - ``InterpolationMethod`` (**None**, '**linear**', '**cubic**')
```

Comment:
Can you also remove the quotes so they don't look like strings? Here and in other symbols as well.